### DISCUSSION TITLE
【提案】Array.prototype.findRight

### DISCUSSION THREAD

**Discussion by Kingwl** [JSCIG/es-discuss#19] [Stage 2]
https://github.com/Kingwl/proposal-array-find-right

**Comment by Kingwl** [comment]
不太清楚接下来需要做啥。。。HELP

**Comment by septs** [comment]
建议去 https://es.discourse.group 发个帖子，问问有没有支持你的。

**Comment by Kingwl** [comment]
https://es.discourse.group/t/array-prototype-findright/538 Thanks!

**Comment by hax** [comment]
等待1月25日TC39会议上讨论。我个人感觉进stage 1应该没有问题。

**Comment by hax** [comment]
已经stage 1，10天后的会议计划进stage 2。
